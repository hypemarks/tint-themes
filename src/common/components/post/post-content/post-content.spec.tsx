import React from 'react';
import { render } from '@testing-library/react';
import PostContent from './post-content';

describe('Test <PostContent /> Component', () => {
  it('should match snapshot', () => {
    const wrapper = render(<PostContent text={'text'} url={'url'} onClick={jest.fn} />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
