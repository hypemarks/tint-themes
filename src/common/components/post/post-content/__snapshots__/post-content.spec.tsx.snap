// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <PostContent /> Component should match snapshot 1`] = `
<DocumentFragment>
  <div
    aria-label="text"
    class="post-content "
    data-testid="post-content"
    style="color: rgb(0, 0, 0); font-family: 'Lato';"
  >
    <div
      class="post-content-link"
      data-testid="post-content-link-button"
      role="button"
    >
      text
    </div>
  </div>
</DocumentFragment>
`;
